### PR TITLE
Allow for overriding of RVM roles

### PR DESCRIPTION
**Pull request by cheald**
It is sometimes useful to limit RVM checks and usage to certain roles. This change simply allows for an `:rvm_roles` variable which may be set to specify the roles that RVM operates on. It defaults to `:all`.

### CODE REVIEW

**Comment by mpapis** [comment]
can you also update README?

**Comment by cheald** [comment]
Yup, can do. Is the `:rvm_roles` key good with you?

**Comment by mpapis** [comment]
sounds good, but I'm not officially maintaining this repo so it's not up to me ;)

**Comment by cheald** [comment]
Ah, heh, of course. :)

File: lib/capistrano/tasks/rvm.rake [hunk 1]
```diff
@@ -4,7 +4,7 @@ RVM_USER_PATH = "~/.rvm"
 namespace :rvm do
   desc "Prints the RVM and Ruby version on the target host"
   task :check do
-    on roles(:all) do
+    on roles( fetch(:rvm_roles, :all) ) do
```
Comment (kirs) [inline lib/capistrano/tasks/rvm.rake]: Could you remove spaces here, please?

**Comment by kirs** [comment]
That would be useful option :+1: Thanks!

File: lib/capistrano/tasks/rvm.rake (hunk 1 above)
Comment (cheald) [inline lib/capistrano/tasks/rvm.rake]: Certainly. Sorry, personal habit.

**Comment by cheald** [comment]
Fixed and squashed.

**Comment by kirs** [comment]
Thanks!

**Comment by mpapis** [comment]
one question, did anyone test it without rvm in one of stages? in rvm1-capistrano3 I had to use an extra check => https://github.com/rvm/rvm1-capistrano3/compare/49b60d9a06ea2cf0e20743ec7724b106fcf82ca1...3e3fb5c375c282479f514cd8a22e0aee2e78539c?w=1 I guess should be updated before #52

**Pull request merged**